ZWPARW/ZWPARW/Object/Leltar.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace ZWPARW.Object
+{
+    // NOTE: Generated code may require at least .NET Framework 4.5 or .NET Core/Standard 2.0.
+    /// <remarks/>
+    [System.SerializableAttribute()]
+    [System.ComponentModel.DesignerCategoryAttribute("code")]
+    [System.Xml.Serialization.XmlTypeAttribute(AnonymousType = true)]
+    [System.Xml.Serialization.XmlRootAttribute(Namespace = "", IsNullable = false)]
+    public partial class Leltar
+    {
+
+        /// <remarks/>
+        [System.Xml.Serialization.XmlElementAttribute("Azonosito")]
+        public HashSet<LeltarAzonosito> azonosito = new HashSet<LeltarAzonosito>();
+    }
+
+    /// <remarks/>
+    [System.SerializableAttribute()]
+    [System.ComponentModel.DesignerCategoryAttribute("code")]
+    [System.Xml.Serialization.XmlTypeAttribute(AnonymousType = true)]
+    public partial class LeltarAzonosito
```

Comment:
1 osztály = 1 fájl, még akkor is ha generátor készíti, vagy ilyen esetben érdemes külön assembly-be tenni, hogy elválljon a generált kód a készítettől.